Anlab.Core/Domain/MailMessage.cs

```diff
@@ -0,0 +1,30 @@
+using System;
+using System.ComponentModel.DataAnnotations;
+using Anlab.Core.Models;
+
+namespace Anlab.Core.Domain
+{
+    public class MailMessage {
+        public MailMessage()
+        {
+            SentAt = DateTime.UtcNow;
+        }
+
+        public int Id { get; set; }
+
+        [Required]
+        public string SendTo { get; set; }
+
+        public bool? Sent { get; set; }
```

Comment:
@jSylvestre my thought with sent was null means not sent, false means permanently not sent (error), and true means successful sent.